cypress/e2e/ModelizerPage/AllView/settings/git.settings.feature

```diff
@@ -0,0 +1,151 @@
+    When I click on "[data-cy=\"project-settings\"]"
+    And I click on "[data-cy=\"git-settings-menu\"]"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-repository-input\"]" text "https://github.com/ditrit/leto-modelizer-project-test"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-username-input\"]" text "test"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-token-input\"]" text "test"
+    And I click on "[data-cy=\"git-form\"] [data-cy=\"git-form-submit\"]"
+
+    Then I expect "positive" toast to appear with text "We have access to your repository 🥳!"
+    And I expect "[data-cy=\"git-form\"]" is closed
+
+    When I click on "[data-cy=\"git-current-branch\"]"
+    Then I expect "[data-cy=\"git-menu-branch-local-master\"]" exists
+    And I expect "[data-cy=\"git-menu-branch-local-master\"]" appear 1 time on screen
+
+    When I click on "[data-cy=\"project-settings\"]"
+    And I click on "[data-cy=\"git-settings-menu\"]"
+    And I click on "[data-cy=\"git-form\"] [data-cy=\"git-form-submit\"]"
+
+    Then I expect "positive" toast to appear with text "We have access to your repository 🥳!"
+    And I expect "[data-cy=\"git-form\"]" is closed
+
+    When I click on "[data-cy=\"git-current-branch\"]"
+    Then I expect "[data-cy=\"git-menu-branch-local-master\"]" exists
+    And I expect "[data-cy=\"git-menu-branch-local-master\"]" appear 1 time on screen
+
+
+  Scenario: Save git configuration twice should not duplicate branches in branch menu
+    When I click on "[data-cy=\"project-settings\"]"
+    And I click on "[data-cy=\"git-settings-menu\"]"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-repository-input\"]" text "https://github.com/ditrit/leto-modelizer-project-test"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-username-input\"]" text "test"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-token-input\"]" text "test"
+    And I click on "[data-cy=\"git-form\"] [data-cy=\"git-form-submit\"]"
+
+    Then I expect "positive" toast to appear with text "We have access to your repository 🥳!"
+    And I expect "[data-cy=\"git-form\"]" is closed
+
+    When I click on "[data-cy=\"git-current-branch\"]"
+    Then I expect "[data-cy=\"git-menu-branch-local-master\"]" exists
+    And I expect "[data-cy=\"git-menu-branch-local-master\"]" appear 1 time on screen
+
+    When I click on "[data-cy=\"project-settings\"]"
+    And I click on "[data-cy=\"git-settings-menu\"]"
+    And I click on "[data-cy=\"git-form\"] [data-cy=\"git-form-submit\"]"
+
+    Then I expect "positive" toast to appear with text "We have access to your repository 🥳!"
+    And I expect "[data-cy=\"git-form\"]" is closed
+
+    When I click on "[data-cy=\"git-current-branch\"]"
+    Then I expect "[data-cy=\"git-menu-branch-local-master\"]" exists
+    And I expect "[data-cy=\"git-menu-branch-local-master\"]" appear 1 time on screen
+
+  Scenario: Expand menu should be closed when menu is open
```

Comment:
should be put in another test file cause only concern TextView. (AND not even git setting/config)

Reply:
done

Reply:
should be in TextView tests...

Reply:
done